src/vivarium_census_prl_synth_pop/utilities.py

```diff
@@ -670,13 +681,28 @@ def get_guardian_duplication_row_counts(
             )
         ]
     )
+    metadata_df["group_rows.row_probability_in_college_group_quarters_under_24"] = len(
+        df.loc[(df["age"] < 24) & (df["housing_type"] == "College")]
+    )
 
     return metadata_df
 
 
 def _get_metadata_counts(
     observer: str, df: pd.DataFrame, location: str, year: int
 ) -> pd.DataFrame:
+    # FIXME: import this from pseudopeople after release
+    COPY_HOUSEHOLD_MEMBER_COLS = {
+        "age": "copy_age",
+        "date_of_birth": "copy_date_of_birth",
+        "ssn": "copy_ssn",
```

Comment:
Should any of these strings be containerized? Super annoying should we ever need to change col names - not to mention reducing chances of typos.

Reply:
These are stored in the right places in pseudopeople. We have circular dependency issues in pseudopeople which prevents an ideal solution but the list are stored in the proper metadata locations.